pom.xml

```diff
@@ -90,6 +90,10 @@
                         </goals>
                     </execution>
                 </executions>
+                <configuration>
```

Comment:
Actually for POC version 8 is fine, but we should also test and validate for 7, coz bpay-api still uses 7 version.